components/push/communications/src/lib.rs

```diff
@@ -246,7 +248,7 @@ impl Connection for ConnectHttp {
             )
             .send()
         {
-            Ok(_) => return Ok(true),
+            Ok(_) => Ok(true),
```

Comment:
Drive-by note: If we set up the `impl_from_error!` chain for push errors, I wonder if we could unify `reqwest::Error` with `CommunicationServerError`, and make this `.send().map(|_| true)`—but that's well outside the scope of this patch.